runtime/cudaq/builder/kernel_builder.cpp

```diff
@@ -479,23 +484,23 @@ QuakeValue applyMeasure(ImplicitLocOpBuilder &builder, Value value,
   return QuakeValue(builder, ret);
 }
 
-QuakeValue mx(ImplicitLocOpBuilder &builder, QuakeValue &qubitOrQreg,
+QuakeValue mx(ImplicitLocOpBuilder &builder, QuakeValue &qubitOrQvec,
```

Comment:
nit: I'm ok with abbreviations and this makes sense to me. Some people like to see names typed out entirely everywhere. Which I think would mean some variation of `qvector` for these. Mentioning it should we document coding style guidelines for runtime, compiler, etc.